[2025-08-18 22:39:03,094] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_branch_dop_operator_v3.condition scheduled__2025-08-19T01:38:00+00:00 [queued]>
[2025-08-18 22:39:03,102] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_branch_dop_operator_v3.condition scheduled__2025-08-19T01:38:00+00:00 [queued]>
[2025-08-18 22:39:03,102] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 22:39:03,102] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2025-08-18 22:39:03,102] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 22:39:03,115] {taskinstance.py:1377} INFO - Executing <Task(BranchPythonOperator): condition> on 2025-08-19 01:38:00+00:00
[2025-08-18 22:39:03,118] {standard_task_runner.py:52} INFO - Started process 2225831 to run task
[2025-08-18 22:39:03,123] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'example_branch_dop_operator_v3', 'condition', 'scheduled__2025-08-19T01:38:00+00:00', '--job-id', '284', '--raw', '--subdir', '/root/Documents/airflow/venv/lib64/python3.9/site-packages/airflow/example_dags/example_branch_python_dop_operator_3.py', '--cfg-path', '/tmp/tmpd9nn30dd', '--error-file', '/tmp/tmpurk0kbou']
[2025-08-18 22:39:03,123] {standard_task_runner.py:80} INFO - Job 284: Subtask condition
[2025-08-18 22:39:03,164] {task_command.py:370} INFO - Running <TaskInstance: example_branch_dop_operator_v3.condition scheduled__2025-08-19T01:38:00+00:00 [running]> on host itoneptl
[2025-08-18 22:39:03,216] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_branch_dop_operator_v3
AIRFLOW_CTX_TASK_ID=condition
AIRFLOW_CTX_EXECUTION_DATE=2025-08-19T01:38:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-19T01:38:00+00:00
[2025-08-18 22:39:03,223] {logging_mixin.py:115} WARNING - /root/Documents/airflow/venv/lib64/python3.9/site-packages/airflow/utils/context.py:290 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-08-18 22:39:03,223] {logging_mixin.py:115} INFO - ------------- exec dttm = 2025-08-19T01:38:00+00:00 and minute = 38
[2025-08-18 22:39:03,223] {python.py:173} INFO - Done. Returned value was: empty_task_1
[2025-08-18 22:39:03,223] {skipmixin.py:140} INFO - Following branch empty_task_1
[2025-08-18 22:39:03,224] {skipmixin.py:175} INFO - Skipping tasks ['empty_task_2']
[2025-08-18 22:39:03,254] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=example_branch_dop_operator_v3, task_id=condition, execution_date=20250819T013800, start_date=20250819T013903, end_date=20250819T013903
[2025-08-18 22:39:03,294] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-18 22:39:03,323] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
